packages/debloat.vm/tools/chocolateyinstall.ps1

```diff
@@ -1,26 +1,41 @@
 $ErrorActionPreference = 'Stop'
 Import-Module vm.common -Force -DisableNameChecking
 
-$packageToolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$packageToolsDir = Join-Path $(pwd) "debloat.vm\tools"
+$packageStartDir = Join-Path $(pwd) "debloat.vm\start"
```

Comment:
Use `Get-Location` instead of `pwd` to make the linter happy:
> Alias can introduce possible problems and make scripts hard to maintain. Please consider changing alias to its full content.